eng/common/scripts/Validate-Package.ps1

```diff
@@ -0,0 +1,291 @@
+#This script is responsible for release preparedness check that's run as part of build pipeline.
+
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $true)]  
+  [string] $PackageName,
+  [Parameter(Mandatory = $true)] 
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $RepoRoot,
+  [Parameter(Mandatory=$True)]
+  [string] $APIKey,  
+  [Parameter(Mandatory=$True)]
+  [string] $ConfigFileDir,
+  [string] $BuildDefinition,
+  [string] $PipelineUrl,
+  [string] $APIViewUri,
+  [string] $Devops_pat = $env:DEVOPS_PAT
+)
+Set-StrictMode -Version 3
+
+. (Join-Path $PSScriptRoot common.ps1)
+. ${PSScriptRoot}\Helpers\ApiView-Helpers.ps1
+. ${PSScriptRoot}\Helpers\DevOps-WorkItem-Helpers.ps1
+
+if (!$Devops_pat) {
+  az account show *> $null
+  if (!$?) {
+    Write-Host 'Running az login...'
+    az login *> $null
+  }
+}
+else {
+  # Login using PAT
+  LoginToAzureDevops $Devops_pat
+}
+
+az extension show -n azure-devops *> $null
+if (!$?){
+  az extension add --name azure-devops
+} else {
+  # Force update the extension to the latest version if it was already installed
+  # this is needed to ensure we have the authentication issue fixed from earlier versions
+  az extension update -n azure-devops *> $null
+}
+
+CheckDevOpsAccess
+
+# Function to validate change log
+function ValidateChangeLog($changeLogPath, $versionString)
+{
+    $validationStatus = [PSCustomObject]@{
+        Name = "Change Log Validation"
+        Status = "Success"
+        Message = ""
+    }
+
+    try
+    {
+        $changeLogFullPath = Join-Path $RepoRoot $changeLogPath
+        Write-Host "Path to change log: [$changeLogFullPath]"        
+        if (Test-Path $changeLogFullPath)
+        {
+            $errOutput = $( $validChangeLog = & Confirm-ChangeLogEntry -ChangeLogLocation $changeLogFullPath -VersionString $versionString -ForRelease $true ) 2>&1
```

Comment:
This line needs a little work. Perhaps you need to make changes to Confirm-ChangeLogEntry